src/avt/Pipeline/Data/avtDataAttributes.h

```diff
@@ -559,6 +564,11 @@ class PIPELINE_API avtDataAttributes
     bool                     GetConstructMultipleCurves() const
                                { return constructMultipleCurves; }
 
+    void                     SetForceRemoveFacesBeforeGhosts(bool doForce)
+                               { forceRemoveFacesBeforeGhosts = doForce; }; 
```

Comment:
Semi colon outside of curly brace not needed here.

Reply:
Done!